content/docs/ui/sending-email/how-to-send-an-email-with-dynamic-transactional-templates.md

```diff
@@ -27,6 +27,12 @@ Before you create and send an email using a dynamic transactional template you n
 4. Select an editor and click **Continue**.
 5. Design your template. For more information on using Handlebars, see [Using handlebars]({{root_url}}/ui/sending-email/using-handlebars/).
 
+##  Unsubscribe modules for dynamic transactional templates
+
+If you want to create a static unsubscribe module for a dynamic transactional template, please review the following:
```

Comment:
Can you provide a brief description of the steps, please? If someone is using a screen reader, they won't be able to decipher the gif. Thanks, @drespinoza!